Cargo.toml

```diff
@@ -5,5 +5,7 @@ version = "0.0.1"
 authors = ["Kenneth Nierenhausen <kennethnierenhausen@gmail.com>"]
 
 [dependencies]
+unix_socket = "0.5.0"
 
-unix_socket = "0.3.0"
+[target.mipsel-unknown-linux-gnu]
```

Comment:
I don't think this is necessary anymore. Your target is `tessel2` with the target JSON file, not `mipsel-unknown-linux-gnu`.